src/comm_manager.cpp

```diff
@@ -369,19 +339,14 @@ void CommManager::send_attitude(void)
 
 void CommManager::send_imu(void)
 {
-  //  if(RF_.sensors_.should_send_imu_data())
-  //  {
+  turbomath::Vector acc, gyro;
+  uint64_t stamp_us;
+  RF_.sensors_.get_filtered_IMU_(acc, gyro, stamp_us);
```

Comment:
Should we have a parameter that lets us turn this feature (filtering the streamed IMU) on or off?